helm/external-dns-app/templates/_helpers.tpl

```diff
@@ -178,9 +183,9 @@ Validate that the provider makes sense
 don't expose that value to the user in the error message.
 */}}
 {{- define "validateValues.provider" -}}
-{{- if and (ne .Values.provider "aws") (ne .Values.provider "azure") (ne .Values.provider "vmware") (ne .Values.provider "inmemory") -}}
+{{- if and (ne .Values.provider "aws") (ne .Values.provider "azure") (ne .Values.provider "gcp") (ne .Values.provider "vmware") (ne .Values.provider "inmemory") -}}
 external-dns: provider
-    Incorrect value provided. Valid values are either 'aws', 'azure' or 'vmware'.
+    Incorrect value provided. Valid values are either 'aws', 'azure', 'gcp' or 'vmware'.
```

Comment:
Seems like 'inmemory' is also a valid value according to line 186

Reply:
this is omitted intentionally, should only be used for testing